src/transcript/TranscriptEvent.ts

```diff
@@ -107,6 +116,24 @@ export class TranscriptEventConverter {
               transcriptAlternative.items.push(transcriptItem);
             }
 
+            for (const entity of alternative.entities) {
+              if (!transcriptAlternative.entities) {
+                transcriptAlternative.entities = [];
```

Comment:
How about setting `entities: []` when you create a TranscriptAlternative object at line 78?

Reply:
I thought about that. But setting entities to line 78 would mean even if the entities are not present, still the alternative object will have an empty entity array. My thought was that if the builder has not requested for content identification or content redaction, the entities are not present in the alternative object that comes from server. Keeping this is mind I added it here so that the TranscriptAlternative will not have entity if it is not asked for.

Please let me know if you have some input on this.

Reply:
Gotcha. `entities` will be an empty array if `alternative.entities` has at least one item.